test/common/testlib.py

```diff
@@ -971,6 +978,14 @@ def _set_emulated_media_theme(self, name: str):
         if self.cdp.browser.name == "chromium":
             self.cdp.invoke("Emulation.setEmulatedMedia", features=[{'name': 'prefers-color-scheme', 'value': name}])
 
+    def _set_direction(self, direction: str):
+        cur_frame = self.cdp.cur_frame
+        if self.is_present("#shell-page"):
+            self.switch_to_top()
+            self.set_attr("#shell-page", "dir", direction)
+        self.switch_to_frame(cur_frame)
+        self.set_attr("html", "dir", direction)
```

Comment:
Is this safe to do when there is no current frame, i.e. the test currently looks at the shell? But anyway, tests are green, and we do have at least the [TestKeys](https://github.com/cockpit-project/pixel-test-reference/compare/2ab940de7ba68a553e5244e634c1db77eff62138..c4c71f2278662fe7c82d76c409f2fa3890c6501c#diff-600b1434b650c24d5e3fafd69e013d648f5dd87d032ac56b1e42905d832be72f) screenshot, which is shell.